app/src/main/res/layout/fragment_contact_us.xml

```diff
@@ -0,0 +1,176 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/constraintLayout_main"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/n_60"
+    tools:context="com.safetyheads.ContactUsFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/constraintLayout_top"
```

Comment:
To jest osobny widok - implementowany w tym tasku -  https://trello.com/c/QQN046tk/25-we-are-hiring